src/site/antora/modules/ROOT/pages/manual/appenders/database.adoc

```diff
@@ -817,6 +817,28 @@ for its format.
 
 **Required**
 
+| [[MongoDbProvider-attr-databaseName]]databaseName
+| 'string'
+|
+|
+It specifies the name of the database for the logger to use.
+
+Overrides the value provided in the connection string if present.
+
+**Required**
+
+| [[MongoDbProvider-attr-collectionName]]collectionName
+| 'string'
+|
+|
+It pecifies the name of the collection for the logger to use.
+
+Overrides the value provided in the connection string if present.
```

Comment:
```suggestion
For backward compatibility, the collection name can also be specified in the
https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongodb-driver-core/com/mongodb/ConnectionString.html[Java-specific connection string]
```

As far as I understand the MongoDB connection string **does not contain** a collection name. The Java driver uses an alternative syntax that can contain a collection name, but we should encourage users to use the [official connection string syntax](https://www.mongodb.com/docs/manual/reference/connection-string/), not the language-specific one.

Reply:
Done. I did add a line to clearify what value will be used if both provided.